bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/RowResultAdapter.java

```diff
@@ -159,6 +162,9 @@ public void cellValue(ByteString newValue) {
      * </ul>
      *
      * A flattened version of the {@link RowCell} map will be sorted correctly.
+     *
+     * <p>In case user applies {@link InterleaveFilter} than a {@link Cell} can appear more than
+     * once, but the duplicate cells will appear one after another.
      */
```

Comment:
than -> then
it's not clear to me, what do you mean 'cells will appear one after another'

I would rephrase: 
<p>Applying {@link InterleaveFilter} may result in a row to contain duplicated cells.

Reply:
`cells will appear one after another` => I mean the duplicate cells will always be in group one after another (in that sequence)

Reply:
no 'but' here.
may be:
> , but the duplicate cells will appear in a group i.e. one after another.

, where duplicates are grouped in sequences.

Reply:
Thanks for catching it 🙏